cgi-bin/Apache/LiveJournal.pm

```diff
@@ -95,7 +95,7 @@ sub handler
             # by the fact that mod_proxy did nothing, requiring mod_proxy_add_forward, then
```

Comment:
Do we still even need this block.

If I understand the difference between `remote_ip` and `client_ip` correctly, this negates the need for us doing this ourselves.